app/assets/javascripts/aspects-sorting.js

```diff
@@ -0,0 +1,15 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
+
+function setupAspectSorting() {
+  $("#aspect_nav .nav").sortable({
+    items: "li.aspect[data-aspect-id]",
+    update: function() {
+      var ordered_aspect_ids = $(this).sortable("toArray", { attribute: "data-aspect-id" }),
+          data = JSON.stringify({ 'ordered_aspect_ids': ordered_aspect_ids });
```

Comment:
Strings must use doublequote.<br>Identifier 'ordered_aspect_ids' is not in camel case.